packages/app/src/containers/Inspector.tsx

```diff
@@ -51,15 +52,29 @@ export default memo(function Inspector() {
             rotation: undefined,
           };
 
-    const views = [
+    const Dimentions = (
       <Fragment key="layout">
         <AlignmentInspector />
         <DimensionsInspector
           {...dimensionsInspectorProps}
           onSetRotation={handleSetRotation}
         />
         <Spacer.Vertical size={10} />
-      </Fragment>,
+      </Fragment>
+    );
+
+    // Only show the text inspector if layer is a text
+    if (selectedLayers.length === 1 && selectedLayers[0]._class === 'text') {
```

Comment:
Let's put the `<TextStyleInspector />` in the same `views` array as all the other inspectors. It wasn't clear from the screenshot/video, cause I collapsed the other sections, but text layers can also have fills, opacity, etc.

The logic Sketch uses is: if at least one text layer is selected, then show the TextStyleInspector. Even if it's 1 text layer and 1 rectangle selected, we still show the TextStyleInspector. E.g. `hasTextLayers && <TextStyleInspector>`

<img width="546" alt="Screen Shot 2021-04-23 at 9 40 41 AM" src="https://user-images.githubusercontent.com/1198882/115903049-f9f94c80-a417-11eb-8cf9-aa646bf376d7.png">